tide/master/control/SceneController.cpp

```diff
@@ -81,12 +81,14 @@ void SceneController::open(const uint surfaceIndex, const QString& uri,
                            const QPointF& coords, BoolCallback callback)
 {
     auto success = false;
-    if (surfaceIndex < _scene.getSurfaces().size())
+    try
     {
         auto loader = ContentLoader{_scene.getGroup(surfaceIndex)};
         success = loader.loadOrMoveToFront(uri, coords);
     }
-
+    catch (...)
+    {
```

Comment:
nothing to log here at least?

Reply:
hmmm yes, I suppose we could log a warning, it could help with debugging.